tests/TestCase.php

```diff
@@ -160,4 +167,31 @@ private function setUpMorphMap()
             'test-model-with-morph-map' => TestModelWithMorphMap::class,
         ]);
     }
+
+    private function setupS3($app)
+    {
+        $s3Configuration = [
+            'driver' => 's3',
+            'key' => getenv('S3_ACCESS_KEY_ID'),
+            'secret' => getenv('S3_SECRET_ACCESS_KEY'),
+            'region' => getenv('S3_BUCKET_REGION'),
+            'bucket' => getenv('S3_BUCKET_NAME'),
+        ];
+
+        $this->canTestS3 = ! (bool) array_search(false, $s3Configuration);
+
+        $app['config']->set('filesystems.disks.s3', $s3Configuration);
+        $app['config']->set(
+            'laravel-medialibrary.s3.domain',
+            'https://'.$s3Configuration['region'].'.amazonaws.com/laravel-medialibrary'
+        );
+
+        if ($this->canTestS3) {
+            register_shutdown_function(function () {
```

Comment:
Couldn't find a way to call `Storage::` or `app('filesystem.disk')` in a ̀tearDownAfterCLass` method, using `register_shutdown_function` is the only working way that I found to clean the S3 bucket